### PR TITLE
Pay 3732 - Site id api retrieve

### PR DESCRIPTION
**Pull request by vigneshpushkaranHMCTS**
### JIRA link ###
[PAY-3695](https://tools.hmcts.net/jira/browse/PAY-3695) - Parent
[PAY-3732](https://tools.hmcts.net/jira/browse/PAY-3732)
[PAY-3731](https://tools.hmcts.net/jira/browse/PAY-3731)

### Description ###
This PR has changes includes populating Legacy sites data(using liquidbase properties) in to database and retrieving the data with new controller.
And this is for to review the implementation.
Thanks in advance!

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: api/src/main/java/uk/gov/hmcts/payment/api/controllers/LegacyReferenceController.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.payment.api.dto.LegacySiteDTO;
+import uk.gov.hmcts.payment.api.model.LegacySite;
+import uk.gov.hmcts.payment.api.service.LegacySiteService;
+
+import java.util.List;
+
+/**
```
Comment (YasharRahvar) [inline api/src/main/java/uk/gov/hmcts/payment/api/controllers/LegacyReferenceController.java:18]: remove all comments, author, version, sinse annotations.

File: api/src/main/resources/application-local.properties [hunk 2]
```diff
@@ -4,10 +4,9 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/payment
 spring.datasource.username=payment
 spring.datasource.password=payment
 spring.datasource.driver=org.postgresql.Driver
-
 spring.liquibase.enabled=${SPRING_LIQUIBASE_ENABLED:true}
 spring.liquibase.change-log=classpath:/db/changelog/db.changelog-master.xml
-
+server.port=8070
```
Comment (YasharRahvar) [inline api/src/main/resources/application-local.properties:9]: what is the reason behind this?

Comment (vigneshpushkaranHMCTS) [reply api/src/main/resources/application-local.properties]: Removed now, Previously added for local port conflict issue.

File: api/src/main/java/uk/gov/hmcts/payment/api/controllers/LegacyReferenceController.java (hunk 1 above)
Comment (vigneshpushkaranHMCTS) [reply api/src/main/java/uk/gov/hmcts/payment/api/controllers/LegacyReferenceController.java]: Removed All the comments

**Comment by vigneshpushkaranHMCTS** [comment]
Closing this PR, Since on new commits Jenkins checks are not running

File: api/src/main/java/uk/gov/hmcts/payment/api/configuration/security/SpringSecurityConfiguration.java [hunk 3]
```diff
@@ -73,7 +73,10 @@ public void configure(WebSecurity web) {
                 "/webjars/springfox-swagger-ui/**",
                 "/swagger-resources/**",
                 "/v2/**",
-                "/refdata/**",
+                "/refdata/methods",
```
Comment (aravindan89) [inline api/src/main/java/uk/gov/hmcts/payment/api/configuration/security/SpringSecurityConfiguration.java:76]: Hi @vigneshpushkaranHMCTS - Why we need to add this separately? Is there any specific reason for this?

Comment (vigneshpushkaranHMCTS) [reply api/src/main/java/uk/gov/hmcts/payment/api/configuration/security/SpringSecurityConfiguration.java:76]: Hi @aravindan89 , The new API method is added to existing controller which by default not have authentication, As discussed with @YasharRahvar @AyanDuttaHMCTS  , We added authentication to only new api to not affect existing API.

**Review by aravindan89 — APPROVED** [review]

**Review by amitchidrewar1301 — APPROVED** [review]

**Comment by vigneshpushkaranHMCTS** [comment]
Closing PR since, continuous-integration/travis-ci/pr  is causing issue

**Pull request closed without merging**